src/api/public/apidocs-new/paths/source_project_name_package_name_history.yaml

```diff
@@ -0,0 +1,86 @@
+get:
+  summary: Get package commit history.
+  description: Get package commit history.
+  security:
+    - basic_authentication: []
+  parameters:
```

Comment:
@eduardoj there is also the `:view` parameter, not sure what it is for though...

Reply:
https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/api/app/controllers/source_controller.rb#L315

Reply:
The `:view` parameter seems not to be used in the backend, for the `/source/$project/$package/_history` path: https://github.com/openSUSE/open-build-service/blob/a3bd82a38dfba6ebef167057bf4da8868cab9a94/src/backend/bs_srcserver#L7640

It also happens with he `:expand` parameter.